app/controllers/concerns/announce_actions.rb

```diff
@@ -40,7 +40,7 @@ def add_announce_filter(action, options)
     end
 
     def new_announce_filter(action, options)
-      filter_options = options.slice(:action, :decorator, :subject)
+      filter_options = options.slice(:action, :decorator, :subject, :resource)
```

Comment:
This is what fixes the announce issue, the new attribute was never getting to the annouce filter